README.md

```diff
@@ -43,3 +43,13 @@ it should be quite adaptable to anyone else's needs.
    `# ./dockhub.py --action remove --dh_user <user name> --dh_group \<group name\>`
 
    Removes \<user name\> from \<group name\>
+
+## For development:
+
+1. Install dev requirements:
+
+   `$ pip install -r requirements-dev.txt`
+
+1. Run tests:
+
+   `$ pytest`
```

Comment:
It doesn't have to be like this. I just kind of threw this together to have something comprehensive. If you have a preferred way of doing things, we can easily change this.

Whatever makes you happy makes me happy.